homeassistant/components/qbus/entity.py

```diff
@@ -10,12 +13,36 @@
 from homeassistant.components.mqtt import ReceiveMessage, client as mqtt
 from homeassistant.helpers.device_registry import DeviceInfo, format_mac
 from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from .const import DOMAIN, MANUFACTURER
+from .coordinator import QbusControllerCoordinator
 
 _REFID_REGEX = re.compile(r"^\d+\/(\d+(?:\/\d+)?)$")
 
 
+def add_new_outputs(
+    coordinator: QbusControllerCoordinator,
+    added_outputs: list[QbusMqttOutput],
+    filter_fn: Callable[[QbusMqttOutput], bool],
+    entity_type: type[QbusEntity],
+    async_add_entities: AddEntitiesCallback,
```

Comment:
```suggestion
    async_add_entities: AddConfigEntryEntitiesCallback,
```
I believe those should be ConfigEntryEntities as well

Reply:
Thank you! Types have been changed now.